src/frequenz/dispatch/_actor_dispatcher.py

```diff
@@ -229,6 +232,7 @@ async def _start_actor(self, dispatch: Dispatch) -> None:
             components=dispatch.target,
             dry_run=dispatch.dry_run,
             options=dispatch.payload,
+            duration=dispatch.duration,
```

Comment:
This is tricky and can be misleading. If you use the high-level interface and you are listening to the start/stop events, or your actor is managed, this duration might not match the time your actor is really running if some dispatches were merged.

I think a better approach would be to include a list of active dispatches, and just pass the full `Dispatch` object there. Another approach would be to pass here a "merged duration" instead of the raw duration of the current dispatch. Depending on the use case one could be more useful than the other.

Reply:
I moved the conversation to #119, I think it is more appropriate to discuss the design in the issue.